doctests.py

```diff
@@ -64,11 +64,11 @@ def command_to_converted_args(location, command):
     insert_arg_index = 2
 
     # Check if running inside venv, if so we need to activate in each process
-    if sys.prefix != sys.base_prefix:
+    if sys.prefix != sys.base_prefix and sys.platform != 'linux':
```

Comment:
As it looks like we only need to activate it on Windows this can be simplified to
```python
# Check if running inside venv on Windows, if so we need to activate in each process
if sys.prefix != sys.base_prefix and os.name == 'nt':
    # Windows path `\` gets dropped, but Windows is happy to accept `/` instead
    activate_script = sys.prefix.replace("\\", "/") + "/Scripts/activate.bat"
```

Reply:
much simpler - thank you